File: RepoTransBench/target_projects/Java/C#/mohak1712_UberUX/public_tests/ExampleInstrumentedPublicTest.cs
using Xunit;

namespace UberUX.Tests.Public
{
    public class ExampleInstrumentedPublicTest
    {
        [Fact]
        public void UseAppContext_Public()
        {
            string packageName = "mohak.uberux.app";
            Assert.Contains("uberux", packageName);
        }
    }
}

File: RepoTransBench/source_projects/Java/mohak1712_UberUX/app/src/test/java/mohak/uberux/ExampleInstrumentedPublicTest.java
package mohak.uberux;

import android.content.Context;
import android.support.test.InstrumentationRegistry;
import android.support.test.runner.AndroidJUnit4;

import org.junit.Test;
import org.junit.runner.RunWith;

import static org.junit.Assert.*;

@RunWith(AndroidJUnit4.class)
public class ExampleInstrumentedPublicTest {
    @Test
    public void useAppContext_public() throws Exception {
        // Just assert that the app context package name contains "uberux"
        Context appContext = InstrumentationRegistry.getTargetContext();
        assertTrue(appContext.getPackageName().contains("uberux"));
    }
}